### PR TITLE
Static HTML

### PR DESCRIPTION
**Pull request by CodeAndWeb**
- built static HTML page
- removed angular app and obsolete files
- created "editors" section

### CODE REVIEW

File: src/public/CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-www.ngx-translate.com
```
Comment (ocombe) [inline src/public/CNAME:1]: don't delete this one, it's used by surge

Comment (CodeAndWeb) [reply src/public/CNAME:1]: Just re-added surge and travis

Comment (ocombe) [reply src/public/CNAME:1]: ok nice

**Comment by CodeAndWeb** [comment]
Sorry... I am not familiar with travis - we are using jenkins on our local servers for our buids...

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+cache: yarn
+
+sudo: false
+
+node_js:
+  - '7'
+
+after_success:
+  - npm run deploy
```
Comment (ocombe) [inline .travis.yml]: can you change this line to:
```
  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then npm run deploy; fi'
```

Comment (CodeAndWeb) [reply .travis.yml]: Like this?

    node_js:
      - '7'

    after_success:
      - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then npm run deploy; fi'

**Comment by ocombe** [comment]
and it's live!! thanks a lot :)

**Pull request merged**